api4/plugin_test.go

```diff
@@ -841,10 +864,40 @@ func TestInstallMarketplacePlugin(t *testing.T) {
 		})
 		pRequest := &model.InstallMarketplacePluginRequest{Id: "com.mattermost.nps", Version: "1.0.3"}
 		plugin, resp := th.SystemAdminClient.InstallMarketplacePlugin(pRequest)
+		CheckInternalErrorStatus(t, resp)
+		require.Nil(t, plugin)
+	})
+
+	t.Run("verify, install and remove plugin", func(t *testing.T) {
```

Comment:
can we also one add test to confirm `uploadplugin` fails when `requiresignature` flag is turned on?